pkg/apis/operator/v1alpha1/apiserver_types.go

```diff
@@ -62,6 +62,9 @@ type APIServerSpec struct {
 	// +optional
 	// Authentication configuration
 	Authentication Authentication `json:"authentication,omitempty"`
+	// LogLevel
+	// +optional
+	LogLevel int `json:"logLevel,omitempty"`
```

Comment:
This doesn't differentiate between set value to `0` and please default me. 

As we discussed 2 weeks ago, if the `LogLevel` is unset (that is `*int` is `null`) in the defaulting webhook we default to the operators `LogLevel`. Otherwise, we leave it as it is.

Reply:
done